packages/mu-local/driver.js

```diff
@@ -61,3 +73,6 @@ module.exports = function createLocalDriver (options) {
     }
   }
 }
+
+module.exports.forceError = function forceError () { ERROR_ON_SEND = true }
```

Comment:
For tests yes - there is a potential silent failure condition in the transport code here: https://github.com/apparatus/mu/blob/master/packages/mu-transport/index.js#L68 the callback result is being ignored and needs to be logged at the least. Added this and then modified local driver to keep coverage numbers. It's not really a vertical configuration for normal use cases - however happy to kill this PR if you have a better idea :)